src/sentry/web/frontend/base.py

```diff
@@ -49,6 +52,70 @@
 audit_logger = logging.getLogger("sentry.audit.ui")
 
 
+class ViewSiloLimit(SiloLimit):
+    def modify_endpoint_class(self, decorated_class: Type[View]) -> type:
+        dispatch_override = self.create_override(decorated_class.dispatch)
```

Comment:
Should we also throw an exception if we determine the class we are modifying already has a `silo_limit` to prevent accidental overrides in subclasses, for instance?  It can result in weird behaviors.

Reply:
That's a good idea. We're missing that from the endpoint decorator as well. I'll update both together.